### PR TITLE
Add RISC-V download page

### PR DESCRIPTION
**Pull request by zprood**
## Done

- Add the RISC-V download page
- Update download/iot page, download/cloud page.

### CODE REVIEW

File: templates/download/iot/index.html [hunk 1]
```diff
@@ -50,62 +50,60 @@ <h2 class="p-heading-icon__title" style="line-height: 4rem;">适用于树莓派
         }}
       </div>
     </section>
-      
-    <section class="p-strip">
+
+    <section class="p-strip is-deep">
       <div class="row">
-        <div class="col-8">
+        <div class="col-2 col-medium-4">
```
Comment (petesfrench) [inline templates/download/iot/index.html:56]:
```suggestion
        <div class="col-2 col-medium-4 u-sv3">
```
To add a little spacing between this heading and the content on medium-small screens

**Comment by petesfrench** [comment]
@zprood It looks like their has been a formatting problem with the individual tab files you might want to look at, apart from that LGTM

File: templates/download/iot/index.html (hunk 1 above)
Comment (zprood) [reply templates/download/iot/index.html]: Thanks, added.

**Comment by zprood** [comment]
> @zprood It looks like their has been a formatting problem with the individual tab files you might want to look at, apart from that LGTM

It's caused by the image size, the images of tab 2 and tab 3, resized the sizes.

File: templates/download/iot/index.html [hunk 2]
```diff
@@ -50,62 +50,60 @@ <h2 class="p-heading-icon__title" style="line-height: 4rem;">适用于树莓派
...
               alt="Intel NUC",
               width="120",
               height="45",
               hi_def=True,
-              loading="lazy",
+              loading="auto",
             ) | safe
           }}
           <p>
             <a href="https://ubuntu.com/download/intel-nuc-desktop">安装Ubuntu桌面版</a>
```
Comment (minkyngkm) [inline templates/download/iot/index.html:71]:
```suggestion
            <a href="https://ubuntu.com/download/intel-nuc-desktop">安装Ubuntu桌面版&nbsp;&rsaquo;</a>
```

File: templates/download/iot/index.html [hunk 3]
```diff
@@ -50,62 +50,60 @@ <h2 class="p-heading-icon__title" style="line-height: 4rem;">适用于树莓派
...
     </section>
-      
-    <section class="p-strip">
+
+    <section class="p-strip is-deep">
       <div class="row">
-        <div class="col-8">
+        <div class="col-2 col-medium-4 u-sv3">
           <h3>支持的平台</h3>
         </div>
```
Comment (minkyngkm) [inline templates/download/iot/index.html:58]: Not a big issue but should this title be in the same line with the items? just wondering!

**Comment by minkyngkm** [comment]
Hi @zprood I just left a comment for one minor thing. If it doesn't have to be changed, it's good to go I think! Nice work!

**Comment by zprood** [comment]
@minkyngkm Thank you. I think the title position is OK.

**Review by minkyngkm — APPROVED** [review]
LGTM

**Pull request merged**